spec/spec_helper.rb

```diff
@@ -11,4 +11,22 @@
 VCR.configure do |config|
   config.ignore_hosts 'codeclimate.com' if ENV['CI']
   config.cassette_library_dir = File.join(ManageIQ::Providers::Vmware::Engine.root, 'spec/vcr_cassettes')
+  config.define_cassette_placeholder(Rails.application.secrets.vmware_infra_defaults[:hostname]) do
+    Rails.application.secrets.vmware_infra[:hostname]
+  end
+  config.define_cassette_placeholder(Rails.application.secrets.vmware_cloud_defaults[:host]) do
+    Rails.application.secrets.vmware_cloud[:host]
+  end
+  config.define_cassette_placeholder(Rails.application.secrets.vmware_cloud_defaults[:userid]) do
+    Rails.application.secrets.vmware_cloud[:userid]
+  end
+  config.define_cassette_placeholder(Rails.application.secrets.vmware_cloud_defaults[:password]) do
+    Rails.application.secrets.vmware_cloud[:password]
+  end
```

Comment:
I don't know if we need cassette placeholders for userid/password since they don't actually end up in the VCR file (just the bearer token does)